script/cluster-up-minikube.sh

```diff
@@ -25,7 +25,7 @@ touch ~/.kube/config
 export KUBECONFIG=$HOME/.kube/config
 export PATH=${PATH}:${GOPATH:?}/bin
 
-MINIKUBE_VERSION=v0.21.0
```

Comment:
why do we have to update minikube version ?

Reply:
it is not necessary, I was just trying to guess why the integration tests where failing. I found the issue. It is regarding the `update` command so I will fix it in a different PR.

Reply:
(Fixed in #352 )